frontend/src/metabase/query_builder/components/view/sidebars/ChartTypeSidebar.styled.tsx

```diff
@@ -9,7 +9,7 @@ export interface OptionRootProps {
 
 const getOptionIconColor = ({ isSelected }: OptionIconContainerProps) => {
   if (isSelected) {
-    return color("white");
+    return color("text-white");
```

Comment:
Use CSS custom property so it's consistent with the change on line 59

```suggestion
    return "var(--mb-color-text-white)";
```